exercises/numbers/isOdd/isOdd.js

```diff
@@ -15,7 +15,12 @@
  */
 function isOdd(num) {
   // Hint: See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Remainder
-  return _____;
+  if (num % 2 === 0){
```

Comment:
This threw me off at first because I had anticipated the first branch to be the "true" case. I'd recommend checking `num % 2 === 1` and reduce the function to

```js
function isOdd(num) {
   return num % 2 === 0;
}
```